library/ssl_tls13_generic.c

```diff
@@ -3224,18 +3193,24 @@ int mbedtls_ssl_generate_application_traffic_keys( mbedtls_ssl_context *ssl, mbe
      * MUST be encrypted under the application traffic key.
      */
 
-    MBEDTLS_SSL_DEBUG_MSG( 3, ( "-->>Calling mbedtls_ssl_tls1_3_make_traffic_keys( ) with the following parameters:" ) );
+    MBEDTLS_SSL_DEBUG_MSG( 3, ( "-->> Calling mbedtls_ssl_tls1_3_make_traffic_keys( ):" ) );
     MBEDTLS_SSL_DEBUG_MSG( 3, ( "-- Hash Algorithm: %s", mbedtls_md_get_name( md_info ) ) );
     MBEDTLS_SSL_DEBUG_MSG( 3, ( "-- Handshake Traffic Secret Length: %d bytes", 
                               mbedtls_hash_size_for_ciphersuite( suite_info ) ) );
+    MBEDTLS_SSL_DEBUG_BUF( 3, "-- Client_traffic_secret:",
```

Comment:
Secrets are usually dumped at log level 5. Example in `library/ssl_tls13_generic.c `:
```
MBEDTLS_SSL_DEBUG_MSG( 5, ( "HKDF Extract -- master_secret" ) );
```

Reply:
Sounds good.